scripts/sync_repo_files.sh

```diff
@@ -7,10 +7,10 @@ set -uo pipefail
 
 git_mail="prometheus-team@googlegroups.com"
 git_user="prombot"
-branch="makefile_common"
-commit_msg="makefile: update Makefile.common with newer version"
-pr_title="Synchronize Makefile.common from prometheus/prometheus"
-pr_msg="Propagating changes from master Makefile.common located in prometheus/prometheus."
+branch="repo_sync"
+commit_msg="Update common Prometheus files"
+pr_title="Synchronize common files from prometheus/prometheus"
+pr_msg="Propagating changes from prometheus/prometheus master."
```

Comment:
```suggestion
pr_msg="Propagating changes from prometheus/prometheus default branch."
```

Just a nit, but I think it sounds a bit better and is future proof :)